samples/algorithms/SimpleVQE.qs

```diff
@@ -0,0 +1,200 @@
+/// # Sample
+/// Simplified Sample of a Variational Quantum Eigensolver
+///
+/// # Description
+/// This is an example of a Variational Quantum Eigensolver (VQE).
+/// This example includes:
+///   1. Simple classical optimization to find minimum of a multi-variable function
+///      in order to find an approximation to the minimum eigenvalue of a hamiltonian
```

Comment:
nitpick: "Hamiltonian" should be capitalized here, I think
```suggestion
///      in order to find an approximation to the minimum eigenvalue of a Hamiltonian
```

Reply:
I could be wrong on this, but if not, it might be worth fixing in the comments throughout.

Reply:
You are right. I'll capitalize this in comments.

Reply:
Updated.